__tests__/client-state/mst.test.ts

```diff
@@ -40,12 +40,13 @@ describe('test the mobx state tree nodes', () => {
   })
 
   function fillOutForm(form: Instance<typeof Form>) {
-    form.fields[0].setValue('20000') // income
-    form.fields[1].setValue('65') //age
-    form.fields[2].setValue(MaritalStatus.SINGLE)
-    form.fields[3].setValue(LivingCountry.CANADA)
-    form.fields[4].setValue(LegalStatus.CANADIAN_CITIZEN)
-    form.fields[5].setValue('true') // Lived in Canada whole life
+    // TODO: this should NOT use numbered indexes to fill the form, as that makes ordering changes cause tests to fail.
```

Comment:
please note this, if it can quickly be fixed, awesome

Reply:
Not super quick, so leaning on finishing up and re-doing these i nthe polish phase or throwing it to the new dev to fix as a layup